src/nxdoc/nuxeo-server/installation/docker-image/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build a Custom Docker Image
+review:
+  date: '2021-04-23'
+  status: ok
+labels:
+  - multiexcerpt
+  - multiexcerpt-include
+toc: true
+description: Discover how to build a custom Docker image.
+tree_item_index: 100
+---
+
+To build an application from Nuxeo, we strongly recommend to customize the Nuxeo Docker Image by building a Docker image from the Nuxeo one. Let's see how you can do this.
+
+## Build a Custom Docker Image From the Nuxeo One
+
+You can simply write a [Dockerfile](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) using the Nuxeo image as [parent image](https://docs.docker.com/glossary/#parent_image).
+A good practice is to use a [build argument](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) before the `FROM` instruction to easily set the version of the Nuxeo parent image, as in the example below:
+
+```Dockerfile
+ARG NUXEO_VERSION=latest
+
+FROM docker.packages.nuxeo.com/nuxeo/nuxeo:${NUXEO_VERSION}
+
+# Execute some commands to add layers on top of the parent image
+```
+
+Then, the custom image can be built by running the following command in the directory of the `Dockerfile`:
+
+```shell
+docker build -t mycompany/myapplication:mytag --build-arg NUXEO_VERSION=10.10 .
+```
+
+Below, you can find some examples of customization that can be done in such a custom Docker image.
+
+## Installing Nuxeo Packages
+
+We provide a utility script to install remote Nuxeo packages from [Nuxeo Connect](https://connect.nuxeo.com/) and local Nuxeo packages when building an image from the Nuxeo image. It can be downloaded from [here](https://github.com/nuxeo/nuxeo/blob/master/docker/install-packages.sh). Once the script is downloaded, move it to the root directory of the Dockerfile. 
```

Comment:
The script should be integrated to the 10.10 HF Docker image with https://jira.nuxeo.com/browse/NXP-30297.